JASP-Desktop/JASP-Desktop.pro

```diff
@@ -8,7 +8,7 @@ message("AM_I_BUILDBOT: '$$[AM_I_BUILDBOT]'")
 COPY_BUILDBOTNESS = $$[AM_I_BUILDBOT] # We need to copy it to make sure the equals function below actually works...
 !equals(COPY_BUILDBOTNESS, "") {
 !equals(COPY_BUILDBOTNESS, "\"\"") { #this should be done less stupidly but I do not want to waste my time on that now
-     GENERATE_LANGUAGE_FILES = true
+	GENERATE_LANGUAGE_FILES = true
```

Comment:
Do we want this on by default?

Reply:
The Build Bot have to compile the po files

Reply:
This is only on when it is buildbot, the normal define is 4 lines up.